sections/git_branch.zsh

```diff
@@ -22,9 +34,31 @@ spaceship_git_branch() {
   local git_current_branch="$vcs_info_msg_0_"
   [[ -z "$git_current_branch" ]] && return
 
+  # Show commit/tag info on detached HEAD
+  if [[ $SPACESHIP_GIT_BRANCH_SHOW_COMMIT == true ]] || [[ $SPACESHIP_GIT_BRANCH_SHOW_TAG == true ]]; then
+    local git_current_branch=$(git symbolic-ref HEAD --short 2> /dev/null)
+  fi
+
   git_current_branch="${git_current_branch#heads/}"
   git_current_branch="${git_current_branch/.../}"
 
+  # Add commit SHA info
+  if [[ $SPACESHIP_GIT_BRANCH_SHOW_COMMIT == true ]]; then
+    local git_commit_info="$SPACESHIP_GIT_BRANCH_PREFIX_COMMIT$SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL"
+    git_commit_info="$git_commit_info$(git_prompt_short_sha)$SPACESHIP_GIT_BRANCH_SUFFIX_COMMIT"
```

Comment:
Where is `git_prompt_short-sha` defined ?

Reply:
Hi @salmanulfarzy!
`git_prompt_short_sha` is defined in `$HOME/.oh-my-zsh/lib/git.zsh`:

```shell
...
# Formats prompt string for current git commit short SHA
function git_prompt_short_sha() {
  local SHA
  SHA=$(command git rev-parse --short HEAD 2> /dev/null) && echo "$ZSH_THEME_GIT_PROMPT_SHA_BEFORE$SHA$ZSH_THEME_GIT_PROMPT_SHA_AFTER"
} 
    
# Formats prompt string for current git commit long SHA
function git_prompt_long_sha() {
  local SHA
  SHA=$(command git rev-parse HEAD 2> /dev/null) && echo "$ZSH_THEME_GIT_PROMPT_SHA_BEFORE$SHA$ZSH_THEME_GIT_PROMPT_SHA_AFTER"
}
...
```

Reply:
This is not an oh-my-zsh theme, your section should work without depending on anything external 😉